package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Typescript/Javascript libs for interacting with Textile APIs.",
   "main": "packages/hub",
   "scripts": {
-    "clean": "lerna clean && lerna run clean",
+    "clean": "npx rimraf dist && lerna clean --yes && lerna run clean",
```

Comment:
faster cleanup and remove this "extra" dist at the top level that is created by the esm build step